chainer/links/connection/lstm.py

```diff
@@ -27,14 +28,31 @@ class LSTM(link.Chain):
         h (chainer.Variable): Output at the previous timestep.
 
     """
-    def __init__(self, in_size, out_size):
+
+    def __init__(self, in_size, out_size,
+                 lateral_init=initializations.orthogonal, upward_init=None,
+                 bias_init=0, forget_bias_init=1):
```

Comment:
Add documentation about `lateral_init`, `upward_init`, `bias_init`, and `forget_bias_init`.